### PR TITLE
[PR1] DEV-42 add MessageTransfer

### PR DESCRIPTION
**Pull request by jmaralo**
This PR adds the MessageTransfer module.

Before merging I'd like to update PacketAdapter, but to prevent conflicts I first need the changes over on [DEV-41](https://github.com/HyperloopUPV-H8/Backend-H8/pull/45), so please check that one first

### CODE REVIEW

**Review by Losina24 — DISMISSED** [review]
Pinta bien

File: src/message_transfer/domain/message.go [hunk 1]
```diff
@@ -1,6 +1,23 @@
 package domain
 
+type id = uint16
+
 type Message struct {
-	Id     uint16
-	Fields map[string]string
+	id     id
+	values map[string]any
```
Comment (smorsua) [inline src/message_transfer/domain/message.go:7]: Why do you used any? Messages only contain strings

File: src/message_transfer/presentation/websocket.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package presentation
+
+import (
+	"github.com/HyperloopUPV-H8/Backend-H8/Shared/server/infra/interfaces"
+	"github.com/HyperloopUPV-H8/Backend-H8/message_transfer/domain"
+)
+
+func MessageRoutine(socket interfaces.WebSocket, messages <-chan domain.Message) {
```
Comment (smorsua) [inline src/message_transfer/presentation/websocket.go:8]: Is the handling of ws.onError done somewhere else?

File: src/message_transfer/domain/message.go (hunk 1 above)
Comment (jmaralo) [reply src/message_transfer/domain/message.go:7]: right now FW says they'll only send strings, but maybe in the future they might send numbers or booleans as well. By using any they can now send whatever they want in a message

Comment (smorsua) [reply src/message_transfer/domain/message.go:7]: Let's stick to the specifications, we can change it in the future

File: src/message_transfer/presentation/websocket.go (hunk 2 above)
Comment (smorsua) [reply src/message_transfer/presentation/websocket.go:8]: Okay

**Review by smorsua — APPROVED** [review]

**Review by lipezaballa — APPROVED** [review]

**Pull request merged**